package.json

```diff
@@ -9,7 +9,7 @@
     "url": "git@github.com:satellytes/satellytes.com.git"
   },
   "engines": {
-    "node": ">= 14.0.0",
+    "node": ">= 16.0.0",
```

Comment:
this will cause yarn to complain when a local node version is lower. let's see who will complain 😁